catalog/views.py

```diff
@@ -115,6 +105,24 @@ def test_func(self):
         return user == self.get_object().owner or user.has_perm("catalog.delete_product")
 
 
+class ProductsByCategoryListView(LoginRequiredMixin, ListView):
+    """Контроллер для категории со списком входящих в нее продуктов"""
+
+    model = Category
+    template_name = "catalog/products_by_category.html"
+    context_object_name = "products"
+
+    def get_queryset(self):
+        category_id = self.kwargs.get("pk")
```

Comment:
ID в этом эндпоинте всегда есть, так что правильнее получать так:
```suggestion
        category_id = self.kwargs["pk"]
```
Ниже тоже

Reply:
Ясно, спасибо, буду знать)